azurerm/resource_arm_container_registry_test.go

```diff
@@ -533,3 +559,51 @@ resource "azurerm_container_registry" "test" {
 }
 `, rInt, location, rInt, sku)
 }
+
+func testAccAzureRMContainerRegistry_networkAccessProfile(rInt int, location string, sku string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "testAccRg-%d"
+  location = "%s"
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "testAccVnet-%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  address_space = [
+    "10.0.0.0/16",
+  ]
+}
+
+resource "azurerm_subnet" "test" {
+  name                 = "testAccSubnet-%d"
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  virtual_network_name = "${azurerm_virtual_network.test.name}"
+  address_prefix       = "10.0.1.0/24"
+}
+
+resource "azurerm_container_registry" "test" {
+  name                = "testAccCr-%d"
```

Comment:
unless our validation is wrong you'll need to remove the `-` here:
```

------- Stdout: -------
=== RUN   TestAccAzureRMContainerRegistry_networkAccessProfile
=== PAUSE TestAccAzureRMContainerRegistry_networkAccessProfile
=== CONT  TestAccAzureRMContainerRegistry_networkAccessProfile
--- FAIL: TestAccAzureRMContainerRegistry_networkAccessProfile (0.88s)
    testing.go:568: Step 0 error: config is invalid: alpha numeric characters only are allowed in "name": "testAccCr-190505182118967719"
FAIL
```

Reply:
Resolved